kubernetes.js

```diff
@@ -181,8 +181,8 @@ const createDeployment = async (project, options) => {
         }
     }
 
-    if (this._app.license.active() && this._cloudProvider === 'openshift') {
-        localPod.spec.securityContext = {}
+    if (this._app.config.driver.options?.podSecurityContext) {
+        localPod.spec.securityContext = this._app.config.driver.options.podSecurityContext
```

Comment:
This would remove the default behaviour if `openshift` is passed as the provider. 

We need to not break existing deployments, because the template.js file has default values.